api/metadata/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
Using this line to comment as it's the first line in a file (and Github still doesn't allow commenting on files as a whole!) 

Unless there's a specific reason not to, I think these test files should all be in `api/tests/metadata/...`?